src/vs/platform/files/electron-browser/diskFileSystemProvider.ts

```diff
@@ -18,7 +18,7 @@ export class DiskFileSystemProvider extends NodeDiskFileSystemProvider {
 		private readonly nativeHostService: INativeHostService,
 		options?: IDiskFileSystemProviderOptions
 	) {
-		super(logService, options);
+		super(logService, undefined, options);
```

Comment:
Shouldn't this pass the args to the super type? This is in the end the file system provider used for desktop VSCode?

Reply:
Ah okay, I am unfamiliar with this code, will fix that.